src/test/unit_tests/test_system_store.js

```diff
@@ -15,6 +15,7 @@ const assert = require('assert');
 const system_store = require('../../server/system_services/system_store').get_instance();
 
 mocha.describe('system_store', function() {
+    this.timeout(90000); // eslint-disable-line no-invalid-this
```

Comment:
Not sure if we shall increase the timeout now. Maybe do it later after optimizations.
Since this code is relevant to MongoDB as well.

Reply:
in mongo it shouldn't affect since it already passes with the default TO of 2 seconds. I prefer it will also pass for postgres. it will be easier to test during development

Reply:
The thing is that we have other tests that are not passing due to timeout.
But okay, we can add timeouts to them lazily.